docs/vitepress-structure.md

```diff
@@ -0,0 +1,104 @@
+  - Base64 encoding of tab data for component consumption
+  - Integration with custom Tabs Vue component via `tabs-data` prop
+  - Extends markdown renderer while preserving original functionality
+
+### Documentation Content
+
+- **index.md**: Landing page configuration using VitePress home layout. Contains:
+  - Hero section with Kyma branding, tagline with phonetic pronunciation, and call-to-action buttons
+  - Main features grid showcasing six key value propositions
+  - Custom `features_additonal` section displaying company adopters/users with logos
+  - Custom CSS classes for styling (`home-kyma-heavy-bold`, `home-kyma-phonetic`)
+  - Asset references for icons and logos with specified dimensions
+- **external-content/**: Aggregates documentation from different repositories.
+- **_sidebar.ts**: Sidebar configuration files imported from the individual modules repositories. Each module has its own sidebar configuration that gets transformed by the `makeSidebarAbsolutePath` function in `config.mjs` to work with the absolute path structure
+
+### Automation and Workflows
+
+- **workflows/deploy.yml**: GitHub Actions workflow that runs every day at midnight to update the documentation. It can also be triggered manually. The CronJob executes the following actions:
+  - Copies content from the documentation repository into the `external-content` folder
+  - Builds and deploys documentation artifacts
+  - Sets up Node.js and installs dependencies
+- **copy_external_content.sh**: Bash script for local development that automates external content aggregation. Process includes:
+  - Clones specific Kyma project repositories
+  - Copies `docs/user` and `docs/assets` folders from each repository to `../docs/external-content/[repo]/docs/`
+  - Provides error handling for missing directories
+  - Automatically cleans up cloned repositories after copying
+  - Target directory structure: `../docs/external-content/[repo-name]/docs/user` and `../docs/external-content/[repo-name]/docs/assets`
+- **move-to-public.sh**: Asset management script that processes files for public distribution. Features:
+  - Scans `./docs/` directory for non-source files using extensive exclusion filters
+  - Excludes common development files (`*.md`, `*.ts`, `*.js`, `*.vue`, `*.css`, config files, etc.)
+  - Skips VitePress internal directories (`.vitepress`, `Node_modules`, `.git`, `.github`)
+  - Copies qualifying files to `./docs/public/` maintaining directory structure
+  - Specifically handles the entire `/assets/` folder copying to the public directory
+  - Provides console output showing file movement operations
+  - Creates necessary target directories automatically
+
+### Configuration and Assets
+
+- **package.json**: Contains scripts for development, build, and preview
+- **public/**: Should contain references to assets that will be propagated on the website (managed by `move-to-public.sh`)
+
+## Building and Previewing Documentation
```

Comment:
I think this should also provide how to build in dev. information are reported here kyma-project/hack/README.md, please think if it makes sense to keep the README file in the hack folder or not (maybe integrate them here)